Sources/Commands/SwiftTestTool.swift

```diff
@@ -837,20 +857,27 @@ extension SwiftTestTool: ToolName {
 
 /// Creates the environment needed to test related tools.
 fileprivate func constructTestEnvironment(
-    sanitizers: EnabledSanitizers,
-    toolchain: UserToolchain
+    toolchain: UserToolchain,
+    options: ToolOptions,
+    buildParameters: BuildParameters
 ) throws -> [String: String] {
+    let sanitizers = options.sanitizers
+
     // This only makes sense for macOS.
```

Comment:
Why?

Reply:
This comment was for the `DYLD_INSERT_LIBRARIES` usage for sanitizers, which is not required on Linux. We should update this comment though since we're using this method for other stuff now.

Reply:
done